src/components/ReportModal/ReportModal.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react';
+import { useNote } from 'components/NoteDataProvider';
+import ReportSection, { PhysicianNotesSection } from './ReportSection';
+import styles from './ReportModal.module.scss';
+import { faUser, faNotesMedical, faRoute, faStickyNote } from '@fortawesome/free-solid-svg-icons';
+import ActionButton from 'components/ActionButton';
+
+interface ReportModalInterface {
+  onConfirm: () => void;
+  onDecline: () => void;
+}
+
+const ReportModal: FC<ReportModalInterface> = ({ onConfirm, onDecline }) => {
+  const { note } = useNote();
+  const reportName = 'Pathway Report';
+  const patientSection = [
+    { name: 'Date', value: note?.date },
+    { name: 'Patient', value: note?.patient },
+    { name: 'Date of Birth', value: note?.birthdate },
+    { name: 'Physician', value: note?.physician }
+  ];
+
+  const observationField = [
+    { name: 'Primary Cancer:', value: undefined },
```

Comment:
Super minor but the colon after the name here makes it show up twice in the actual modal